server/src/main/java/io/envoyproxy/controlplane/server/DefaultExecutorGroup.java

```diff
@@ -0,0 +1,22 @@
+package io.envoyproxy.controlplane.server;
+
+import com.google.common.util.concurrent.MoreExecutors;
+
+import java.util.concurrent.Executor;
+
+/**
+ * Default implementation of {@link ExecutorGroup} which
+ * always returns {@link com.google.common.util.concurrent.MoreExecutors#directExecutor()}.
```

Comment:
supernit: shorten this to reference `MoreExecutors#directExecutor` instead of the fqn